.travis.yml

```diff
@@ -9,20 +9,40 @@ env:
     - MINIKUBE_HOME=$HOME
     - MINIKUBE_VER=v0.28.2
     - K8S_VER=1.11.2
-    - KUBEVIRT_VER=v0.8.0-alpha.1
+    - KUBEVIRT_VER=v0.8.0
 
-matrix:
+jobs:
   include:
-  - name: Check syntax of all templates
-    env: TARGET=syntax-tests
-  - name: Functional test of the Fedora template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=fedora28
-  - name: Functional test of the Ubuntu template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=ubuntu1804
-#  - name: Functional test of the OpenSUSE template
-#    env: TARGET=functional-tests TEST_FUNCTIONAL=opensuse15
-  - name: Functional test of the CentOS/RHEL template
-    env: TARGET=functional-tests TEST_FUNCTIONAL=rhel75
+    - stage: "Tests"
+      name: Check syntax of all templates
+      env: TARGET=syntax-tests
+      # -e is important to let make use the vars defined in "matrix:" above
+      script: make -e $TARGET
+    - script: make -e $TARGET
+      name: Functional test of the Fedora template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=fedora28
+    - script: make -e $TARGET
+      name: Functional test of the Ubuntu template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=ubuntu1804
+    - script: make -e $TARGET
+      name: Functional test of the CentOS/RHEL template
+      env: TARGET=functional-tests TEST_FUNCTIONAL=rhel75
+    - stage: "Deploy"
+      name: deploy 
+      before_script: skip
+      script: skip
+      before_deploy:
+      - cat presets/*.yaml templates/*.yaml > ./all.yaml
```

Comment:
This does not work, as the ymals need to be divided by `---`.

`make common-templates.yaml` can now do the job: https://github.com/fabiand/common-templates/blob/master/makefile#L30

Reply:
fixed